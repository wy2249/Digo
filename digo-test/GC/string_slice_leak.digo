func digo_main() void {
    s := []string{}

    c := "23123423"
    e := "324123423"
    s = append(s, c)
    s = append(s, e)

    fu := s[1]

    println("%s", fu)

    fu = s[0]
    println("%s", fu)

}
