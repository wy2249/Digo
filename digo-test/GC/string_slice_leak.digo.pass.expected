324123423
23123423
